shared/charts/backup-exporter/templates/configmap.yaml

```diff
@@ -15,20 +15,20 @@ data:
     INTERMEDIATE_BUCKET={{ .Values.localIntermediateBucket }}
     LOG_BUCKET={{ .Values.logBucket }}
 
-    gsutil -q ls gs://${DESTINATION_BUCKET}/ 2>1 >/dev/null || { echo "Destination bucket is not accessible"; exit 1; }
-    gsutil -q ls gs://${INTERMEDIATE_BUCKET}/ 2>1 >/dev/null || { echo "Intermediate bucket is not accessible"; exit 1; }
-    gsutil -q ls gs://${LOG_BUCKET}/ 2>1 >/dev/null || { echo "Log bucket is not accessible"; exit 1; }
+    gsutil -q ls "gs://${DESTINATION_BUCKET}/" 2>1 >/dev/null || { echo "Destination bucket is not accessible"; exit 1; }
```

Comment:
I noticed the `2>1`... this will redirect stderr to a file called `1`. Perhaps what you wanted is `2>&1`?

Reply:
+ 2 more times below

Reply:
thanks @stepanstipl, fixed.